src/core/Expander/ExpanderContent/ExpanderContent.baseStyles.tsx

```diff
@@ -6,51 +6,49 @@ import { padding } from '../../theme/utils';
 import { ExpanderContentProps } from './ExpanderContent';
 
 export const baseStyles = withSuomifiTheme(
-  ({ theme }: TokensAndTheme & Partial<ExpanderContentProps>) => {
-    return css`
-      ${element({ theme })}
-      ${font({ theme })('bodyText')}
+  ({ theme }: TokensAndTheme & Partial<ExpanderContentProps>) => css`
+    ${element({ theme })}
+    ${font({ theme })('bodyText')}
       background-color: ${theme.colors.whiteBase};
```

Comment:
Indentation is off here too with `background-color`. Can we do something to prettier to fix this?

Reply:
I assume we cannot unless we create some custom formatting rules which I wouldn't do.

Reply:
Seems to be related to styles in "main" level of baseStyles with two adjanced template literals without terminating semicolons. 😓

Reply:
Searched through all references of `({ theme })` and fixed indentation issues. It's possible that some still remain, but got at least quite a few.

Reply:
This should be also now fixed.